### PR TITLE
Delete deployment freezes

### PR DESCRIPTION
**Pull request by tothegills**

### CODE REVIEW

**Review by mik-ky — APPROVED** [review]

File: source/Octopus.Server.Client/IOctopusAsyncClient.cs [hunk 1]
```diff
@@ -796,6 +796,28 @@ public interface IOctopusAsyncClient : IDisposable
         /// <returns>A task resource that provides details about the background task that deletes the specified resource.</returns>
         Task Delete(string path, object pathParameters, object resource, CancellationToken cancellationToken);
 
+        /// <summary>
+        /// Deletes the resource at the given URI from the server using a the DELETE verb. Deletes in Octopus happen
+        /// asynchronously via a background task
+        /// that is executed by the Octopus Server. The payload returned by delete will be the task that was created on the
```
Comment (domenicsim1) [inline source/Octopus.Server.Client/IOctopusAsyncClient.cs:802]: is it really?

File: source/Octopus.Server.Client/Model/DeploymentFreezes/DeleteDeploymentFreezeResponse.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Octopus.Client.Model.DeploymentFreezes;
+
+public class DeleteDeploymentFreezeResponse
+{
+
```
Comment (domenicsim1) [inline source/Octopus.Server.Client/Model/DeploymentFreezes/DeleteDeploymentFreezeResponse.cs:5]: We say we return a task but this returns nothing, am I missing something here?
Is this something the Async client takes care of for us?

**Review by domenicsim1 — COMMENTED** [review]
Unsure about some stuff, see questions.

File: source/Octopus.Server.Client/Model/DeploymentFreezes/DeleteDeploymentFreezeResponse.cs (hunk 2 above)
Comment (tothegills) [reply source/Octopus.Server.Client/Model/DeploymentFreezes/DeleteDeploymentFreezeResponse.cs:5]: Thanks Dom, the comment is incorrect!

**Pull request merged**